docs/core/install/linux-snap.md

```diff
@@ -130,6 +130,29 @@ The certificate location will vary by distro. Here are the locations for the dis
 | **OpenSUSE** | `/etc/ssl/ca-bundle.pem`                            |
 | **Solus**    | `/etc/ssl/certs/ca-certificates.crt`                |
 
+## Troubles resolving dotnet
+
+It's common for other apps, such as the OmniSharp extension for Visual Studio Code, to try to resolve the location of the .NET SDK. Typically this done by figuring out where the `dotnet` executable is located. A snap-installed .NET SDK may confuse these apps. When these apps can't resolve the .NET SDK, you'll see an error similar to one of the following messages:
```

Comment:
Everything looks great and helpful. Just small thing here. The second sentence should start with "Typically, this is done...". Notice the comma and the addition of the word "is".